### PR TITLE
Add  key to configuration to fix #1022

### LINKED ISSUE #1022: Unrecognized key(s) in object: '$schema'
```
Chromatic CLI v11.6.0
https://www.chromatic.com/docs/cli


✖ Configuration file chromatic.config.json was invalid, please check the allowed keys.

- Unrecognized key(s) in object: '$schema'



→ View the full stacktrace below

If you need help, please chat with us at https://www.chromatic.com/docs/cli for the fastest response.
You can also email the team at support@chromatic.com if chat is not an option.

Please provide us with the above CLI output and the following info:
{
  "timestamp": "2024-08-12T22:31:41.116Z",
  "sessionId": "aae7d813-d95a-414a-8356-8aec5a669236",
  "nodePlatform": "darwin",
  "nodeVersion": "20.16.0",
  "packageName": "chromatic",
  "packageVersion": "11.6.0",
  "flags": {
    "outputDir": [],
    "storybookBuildDir": [],
    "externals": [],
    "onlyStoryFiles": [],
    "onlyStoryNames": [],
    "untraced": [],
    "fileHashing": true,
    "interactive": true,
    "appCode": []
  },
  "errorType": "Error",
  "errorMessage": "✖ Configuration file chromatic.config.json was invalid, please check the allowed keys."
}

Error: ✖ Configuration file chromatic.config.json was invalid, please check the allowed keys.

- Unrecognized key(s) in object: '$schema'
- ```

### PR DESCRIPTION
**Pull request by winkerVSbecks**
We now offer the option to [validate the config file using JSON schema](https://www.chromatic.com/docs/cli/#chromatic-config-file). This PR allows the user to specify the schema within the config file itself. 

Fixes #1022

<details>
  <summary>📦 Published PR as canary version: <code>11.7.1--canary.1023.10387892202.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install chromatic@11.7.1--canary.1023.10387892202.0
  # or 
  yarn add chromatic@11.7.1--canary.1023.10387892202.0
  ```
</details>

### CODE REVIEW

**Review by thafryer — APPROVED** [review]

**Comment by winkerVSbecks** [comment]
@thafryer I don't have merge access. Could you merge this in when you get a chance. Thank you

**Review by skitterm — APPROVED** [review]

**Comment by skitterm** [comment]
@winkerVSbecks I'm seeing a [test failing](https://app.circleci.com/pipelines/github/chromaui/chromatic-cli/3905/workflows/bfca1654-36e9-4cde-a4eb-d04116d18854/jobs/3601) that is preventing the merge queue from merging this PR. Are the tests passing for you locally? This test has [failed before](https://app.circleci.com/pipelines/github/chromaui/chromatic-cli/3897/workflows/688aac60-f948-47ef-8314-26d0d9c73c9c/jobs/3594), but rerunning the CI build a couple times didn't resolve it for me.

**Comment by winkerVSbecks** [comment]
@skitterm `trace.test.ts` is failing for me locally. But not this one. I'm not sure why either are failing. This is my first contribution to this codebase.

**Comment by winkerVSbecks** [comment]
@skitterm actually all tests pass locally now. I just hadn't run a build so trace tests were failing.

**Comment by ghengeveld** [comment]
:rocket: PR was released in [`v11.7.1`](https://github.com/chromaui/chromatic-cli/releases/tag/v11.7.1) :rocket:

**Pull request merged**
